assets/security.txt

```diff
@@ -0,0 +1,3 @@
+Contact: admin@oppia.org
+Preferred-Languages: en
+Canonical: https://www.oppia.org/security.txt
```

Comment:
Add an `Acknowledgements:` section that points to `https://www.oppia.org/about#credits`

Reply:
Updated, ptal!